DemoApp/DemoApp/DemoApp/DTOs/StudentDtoWithoutId.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DemoApp.DTOs
+{
+    public class StudentDtoWithoutId
+    {
+        public int Grade { get; set; }
```

Comment:
Why only grade?

Reply:
added a new dto which include person details when a student is requested